### PR TITLE
98 refactor uvr to constraint

### PR DESCRIPTION
**Pull request by GD-ResilienX**

### CODE REVIEW

**Comment by GD-ResilienX** [comment]
Note that the github webviewer diff is atrocious. Open up the original and new versions of the file in notepad++ or vscode or something to get a side-by-side comparison. The changes should be contained to simply updating every 'uvr' string with 'constraint'.

File: ConstraintApi.yaml [hunk 1]
```diff
@@ -1,1156 +1,1152 @@
...
-      properties:
-        reference:
-          $ref: '#/components/schemas/ConstraintReference'
-        details:
-          $ref: '#/components/schemas/ConstraintDetails'
+---
+openapi: 3.0.1
+info:
+  title: Constraint Reporting API
+  description: 'Volume restrictions can be issued for many reasons, and this API serves as a gateway for being able to properly submit them to the appropriate FIMS'
```
Comment (mtthwcmpbll) [inline ConstraintApi.yaml:5]: I see that we still use Volume3D/4D as our data schemas which makes sense (we're defining a volume), but is it still appropriate to call this a volume restriction here?  It feels like a reasonable synonym, but maybe just "Constraints can be issued for many reasons".

**Review by mtthwcmpbll — CHANGES_REQUESTED** [review]
Looks good, left one question / suggested change.

File: ConstraintApi.yaml (hunk 1 above)
Comment (GD-ResilienX) [reply ConstraintApi.yaml:5]: Yeahhhhh, I did intentionally leave this out as it seemed like an unnecessary extra change, but I think any reasoning I had for it is defeated by literally anyone noticing it :D

**Review by mtthwcmpbll — APPROVED** [review]

**Review by aliciatitus — APPROVED** [review]

**Pull request merged**